### PR TITLE
[OBSDEF-6074] - Added antiaffinity rules for objectscale-portal

### PR DESCRIPTION
**Pull request by maheshkulkarni4**
## Purpose
[OBSDEF-6074](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-6074)
Added Pod Anttiaffinity rules for ObjectScale Portal, so that two pods of objectscale-portal wont deploy on same node. 

## PR checklist
- [x] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
_Paste URL of charts-custom-ci job here_

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: objectscale-portal/templates/objectscale-portal-deployment.yaml [hunk 1]
```diff
@@ -56,13 +55,27 @@ spec:
...
+    {{- if eq .Values.enableAntiAffinity true }}
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                app.kubernetes.io/component: objectscale-portal
+                app.kubernetes.io/name: objectscale-manager
+            namespaces:
+            - dellemc-uiprim78-domain-c44
```
Comment (Bjohnson131) [inline objectscale-portal/templates/objectscale-portal-deployment.yaml:68]: ? pretty sure this isn't right..

Comment (eddiepavkovic) [reply objectscale-portal/templates/objectscale-portal-deployment.yaml:68]: remove hardcoded namespaces.

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]
remove hard coded namespaces.

File: objectscale-portal/templates/objectscale-portal-deployment.yaml (hunk 1 above)
Comment (SarahAFochs) [inline objectscale-portal/templates/objectscale-portal-deployment.yaml:68]:
```suggestion
            - {{ .Release.Namespace }}
```

File: objectscale-portal/templates/objectscale-portal-deployment.yaml [hunk 2]
```diff
@@ -37,7 +37,6 @@ spec:
 {{ include "common-lib.labels-sp-integrated-all" . | indent 8}}
       annotations:
 {{ include "common-lib.vsphere-emm-integrated_annotation" . | indent 8}}
-    spec:
```
Comment (vgupta-mickey) [inline objectscale-portal/templates/objectscale-portal-deployment.yaml:40]: moved spec at wrong place. Need to fix it. I am going to fix it for you and push the changes.

**Review by vgupta-mickey — APPROVED** [review]

**Review by vgupta-mickey — APPROVED** [review]
I fixed your spec. Please test now.

File: objectscale-portal/templates/objectscale-portal-deployment.yaml (hunk 1 above)
Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-deployment.yaml]: Done. Please review.

Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-deployment.yaml]: Done

[the remark this answers is not in the document]
Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-deployment.yaml]: Done

**Pull request merged**